src/SessionOne.java

```diff
@@ -5,50 +5,73 @@ public static void main(String[] args) {
         Scanner input = new Scanner(System.in);
         System.out.println("Press 1, if you want to see if it's an SessionOne number");
         System.out.println("Press 2, whether you want to see if it's an Strong number");
+
         int option = input.nextInt();
+
         System.out.println("Enter a number");
         String number = String.valueOf(input.nextInt());
 
-            switch (option) {
-                case 1:
-                    isAnAmstrongNumber(number);
-                case 2:
-                    isAnStrongNumber(number);
-            }
+        switch (option) {
+            case 1:
+                isAnAmstrongNumber(number);
+            case 2:
+                isAnStrongNumber(number);
+            case 3:
+                isAPerfectNumber(Integer.valueOf(number));
         }
+    }
 
-        public static void isAnAmstrongNumber(String number) {
-            String[] digits = number.split("(?<=.)");
-            double is_amstrong = 0;
-            for (int i = 0; i < digits.length; i++) {
-                is_amstrong += Math.pow(Double.parseDouble(digits[i]), 3);
-            }
-            if (Integer.valueOf(number) == is_amstrong) {
-                System.out.println("it is an SessionOne number");
-            } else {
-                System.out.println("it is not an SessionOne number");
-            }
+    public static void isAnAmstrongNumber(String number) {
+        String[] digits = number.split("(?<=.)");
+
+        double is_amstrong = 0;
+
+        for (int i = 0; i < digits.length; i++) {
+            is_amstrong += Math.pow(Double.parseDouble(digits[i]), 3);
```

Comment:
Stick to one convention, otherwise your code seems sloppy, it may seem trivial, but your coding quality will also be measured by it.

```suggestion
            isAmstrong += Math.pow(Double.parseDouble(digits[i]), 3);
```

Reply:
Oppsie, you have told me that before...